pkg/remotewrite/config.go

```diff
@@ -259,6 +277,16 @@ func parseArg(text string) (Config, error) {
 			if err := c.TrendAsNativeHistogram.UnmarshalText([]byte(v)); err != nil {
 				return c, fmt.Errorf("trendAsNativeHistogram value must be true or false, not %q", v)
 			}
+
+		// TODO: add the support for trendStats
+		// strvals doesn't support the same format used by --summary-trend-stats
+		// using the comma as the separator, because it is already used for
+		// dividing the keys.
+		//
+		//if v, ok := params["trendStats"].(string); ok && len(v) > 0 {
+		//c.TrendStats = strings.Split(v, ",")
+		//}
+
```

Comment:
@na-- Ideas? Should we use another char (e.g. `|`)?

```
./k6 -o prometheus-remote-write url=http://promurl/api/v1/write,trendStats=p(99)|avg
```

WDYT?

Reply:
hmm no, sorry, but this might be a good time to re-evaluate if we want to use something like strvals at all :thinking: 

IIRC this was one of the reason that the influxdb output decodes text parameters as a URL, though that also has its issues :disappointed:

Reply:
we can postpone this decision by not supporting any text-based configuration at the beginning :thinking: just environment variables will probably be sufficient, for now?

Reply:
https://github.com/grafana/xk6-output-prometheus-remote/pull/70